Include/cpython/pyatomic_msc.h

```diff
@@ -912,6 +912,30 @@ _Py_atomic_store_ptr_release(void *obj, void *value)
 #endif
 }
 
+static inline void
+_Py_atomic_store_int_release(int *obj, int value)
+{
+#if defined(_M_X64) || defined(_M_IX86)
+    *(int volatile *)obj = value;
+#elif defined(_M_ARM64)
+    __stlr32((unsigned __int32 volatile *)obj, (unsigned __int32)value);
```

Comment:
It terms of memory alignment, is it safe to cast `int*` to `int32*`? Is it the same size?

Please add:

```
_Py_atomic_ASSERT_ARG_TYPE(unsigned __int32);
```